tests/libcxx/expected_results.txt

```diff
@@ -504,7 +504,7 @@ std/strings/char.traits/char.traits.specializations/char.traits.specializations.
 std/strings/char.traits/char.traits.specializations/char.traits.specializations.wchar.t/move.pass.cpp FAIL
 
 # C++20 P1135R6 "The C++20 Synchronization Library"
-std/atomics/types.pass.cpp FAIL
+std/atomics/types.pass.cpp PASS

```

Comment:
If this passes, it should simply be removed from `expected_results.txt` (anything not mentioned here is expected to `PASS`) and it should also be removed from `skipped_tests.txt` which is the Microsoft-internal counterpart to this file.